### PR TITLE
Prepare for Conan center inclusion

### PR DESCRIPTION
**Pull request by uilianries**
/cc @danimtb

### CODE REVIEW

**Comment by theirix** [comment]
Thank you for the mingw build!

About the license - libev have a dual license, with BSD 2-Clause and GPL (see the last paragraph in their license file). I think we should refer to an original license or (maybe) use a license list (do not know if it is allowed for bintray).

**Comment by uilianries** [comment]
I read that sentence, they said BSD-2 OR GPL. I think dual license (using a list) will be better. I gonna update this. thanks for reviewing!

**Review by theirix — APPROVED** [review]
Looks good, thanks!

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ matrix:
         env: CONAN_GCC_VERSIONS=7 CONAN_DOCKER_IMAGE=conanio/gcc7
       - <<: *linux
         env: CONAN_GCC_VERSIONS=8 CONAN_DOCKER_IMAGE=conanio/gcc8
+      - <<: *linux
+        env: CONAN_GCC_VERSIONS=8 CONAN_DOCKER_IMAGE=conanio/gcc9
```
Comment (theirix) [inline .travis.yml]: Should be CONAN_GCC_VERSIONS=9

**Review by danimtb — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (uilianries) [reply .travis.yml]: good catch!

**Pull request merged**